applications/ConvectionDiffusionApplication/python_scripts/coupled_structural_thermal_solver.py

```diff
@@ -47,8 +47,8 @@ def GetDefaultParameters(cls):
                 "material_import_settings": {
                         "materials_filename": "ThermalMaterials.json"
                 }
```

Comment:
You missed a `,` here

Reply:
Thanks for spotting. Since I do not run in local, I did not see this. My bad